### DISCUSSION TITLE
How to run CodeCepts Scripts for Microsoft Edge browser.

### DISCUSSION THREAD

**Discussion by Arthanarieaswaran** [codeceptjs/CodeceptJS#3647] [Q&A]
Hi All,

I would like to run my CodeCepts scripts on Microsoft Edge browser, kindly let me know the configuration and sample configuration code.

Thanks and Regards
Artha

**Comment by adudyak** [comment]
exports.config = {
  // Other configuration options...

  helpers: {
    Playwright: {
      // Other Playwright options...

      browser: 'chromium',
      channel: 'msedge',
      executablePath: '/usr/bin/microsoft-edge' //path to .exe of edge
    }
  }
};

Sry for formatting, using tablet. Edge uses same chromium engine, as chrome, so you can use defaul chromium setting, unless want to test some code, triggered specifically for edge

**Comment by myrepojuly** [comment]
@Arthanarieaswaran If you are using `Webdriver`, then in the `codecept.conf.js` file use   `browser: 'MicrosoftEdge'`
